BLE_GattServer_FOTAService/MCUboot/README.md

```diff
@@ -0,0 +1,163 @@
+# FOTA service MCUboot example
+
+This application extends the [Mock](../Mock) example by including a bootloader based on MCUboot.
+
+## Usage
```

Comment:
Can you add a section with the limitations as of today (limited board support for example)

Reply:
The section on **Hardware requirements** mentions that the example only works on nRF52840_DK. Personally, I would like to attempt it again on my B_L4S5I_IOT01A. If that fails, I'll add a section on limitations...